metals/src/main/scala/scala/meta/metals/Configuration.scala

```diff
@@ -43,14 +43,14 @@ object Configuration {
       enabled: Boolean = true,
       onSave: Boolean = false,
       version: String = "1.4.0",
-      confPath: Option[RelativePath] = None
+      confPath: Option[RelativePath] = Some(Scalafmt.defaultConfPath)
```

Comment:
These defaults were set in the vscode extension and I thought that this should work fine if the configs are absent. What is the advantage of setting it to `None`?

Reply:
It's a conceptual argument. Scalafix and Scalafmt don't require a configuration, so I would prefer letting them decide what to do in case the config is absent.

Under this spirit, I would rather remove the defaults from the VSCode extension.

Reply:
I think it makes sense to distinguish between "no configured file" vs. "user wants to use this particular config file"

Reply:
These two features are enabled now (as considered stable), but if the config location is not set, it will require an extra action from user. I agree with @gabro [from the past](https://github.com/scalameta/metals/issues/16#issuecomment-343094392):

> I would just like to point out that when using language-server scalafix is already included. The user just needs to provide a .scalafix.conf and it starts working. No additional installation is required, other than the semanticdb-scalac compiler plugin.

Also I don't think that removing default locations from vscode is good, because currently it gives user a clue where they are expected to put those files. 

----

@olafurpg I didn't get, what is your point?

Reply:
I'm fine either way, probably best to set a default value instead of `None`. I agree with @gabro that conceptually it's good to leave to scalafix/scalafmt to decide what to do on "empty configuration" but having a default value make it clearer for users what goes into that setting.

Reply:
OK, so I left the defaults. I just want to note that in case of Atom, users won't be able to set it to `None`, only to `Some("")`, so conceptual differences aside, I wouldn't use `Option` in this config.